utils/js/LangUtils.ts

```diff
@@ -19,6 +19,7 @@ import {
     uniq,
     uniqBy
 } from 'lodash';
+import {v4} from 'uuid';
```

Comment:
Nice to have this as a util. 👍  We should add `uuid` to hoist-react `package.json` so it's an explicit dependency. Right now I think we're getting it as a transitive dev dependency only:

```
yarn why uuid
yarn why v1.22.19
[1/4] 🤔  Why do we have the module "uuid"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "uuid@8.3.2"
info Reasons this module exists
   - "@xh#hoist-dev-utils#webpack-dev-server#sockjs" depends on it
   - Hoisted from "@xh#hoist-dev-utils#webpack-dev-server#sockjs#uuid"
info Disk size without dependencies: "332KB"
info Disk size with unique dependencies: "332KB"
info Disk size with transitive dependencies: "332KB"
info Number of shared dependencies: 0
✨  Done in 0.18s.
```

Reply:
Good call, meant to do this already. Updated the PR.  Also moved this method from `LangUtils` to `FetchService.generateCorrelationId(prefix?: string)` for various reasons.  But could keep it in `LangUtils` if you think it'd be useful in other contexts.